### PR TITLE
Add ability to process filepaths containing the @ sign

### PR DESCRIPTION
**Pull request by masaball**

### CODE REVIEW

File: lib/file_locator.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ def location
     when 's3'
       S3File.new(uri).object.presigned_url(:get)
     when 'file'
-      Addressable::URI.unencode(uri.path)
+      Addressable::URI.unencode(uri.to_s.gsub(/file:\/\//, ''))
```
Comment (cjcolvar) [inline lib/file_locator.rb:54]: Could this also use `sanitize_uri`?
```suggestion
      Addressable::URI.unencode(ActiveEncode.sanitize_uri(uri))
```

**Review by cjcolvar — COMMENTED** [review]
This looks great!  Do you know if the MediaConvert adapter needs any changes as well?

File: lib/file_locator.rb (hunk 1 above)
Comment (masaball) [reply lib/file_locator.rb:54]: Good catch. Missed that one when consolidating everything else.

**Comment by masaball** [comment]
I added the file to the MediaConvert tests for filenames with special characters and those tests are passing so I think it is good.

**Review by cjcolvar — APPROVED** [review]
Looks good!

**Pull request merged**